docs/readmes/cwf/dev_testing.md

```diff
@@ -57,10 +57,29 @@ Runs a UE simulator service and all tests.
 
 Runs an iperf3 server to drive traffic through CWAG.
 
+#### Entire test suite
+
 To run all setup work and the entire CWF integration test suite, run
 
 ```bash
 [HOST] fab integ_test
 ```
 
+Once the above command has been executed, which means that the set-up of the VMs etc. has been
+performed, command-line options can be utilized to rerun the tests without redoing the set-up
+
+```bash
+[HOST] fab integ_test:provision_vm=False,no_build=True
+```
+
+#### Individual tests
+
+The command above can be further modified to run one integration test at a time
+
+```bash
+[HOST] fab integ_test:provision_vm=False,no_build=True,skip_unit_tests=True,test_re=<TEST_TO_RUN>
```

Comment:
`gotestsum --format=standard-verbose --rerun-fails=0 --packages='./...' --junitfile ../True  -- -test.short -timeout 50m -count 1 -tags=all `-test_re=<TESTNAME>` is about 4 times faster for single tests so I would add this command to the readme (as well?)

Reply:
Would `gotestsum` need to be specifically installed on the host?

Reply:
@crasu I've pushed this commit https://github.com/magma/magma/pull/13111/commits/7059af0a0dda4b7192eee5bcfc33bc95441e9fa0 to add instructions on how to run it with `gotestsum`.

@Neudrino The instructions I've added specify that you should execute `gotestsum` on the CWAG test VM, where it is already available.